src/main/java/com/sarality/list/NonScrollingListViewInitializer.java

```diff
@@ -39,6 +39,11 @@ public NonScrollingListViewInitializer(Activity activity, LinearLayout listView,
     return this;
   }
 
+  public NonScrollingListViewInitializer<T, H> withEmptyList(View emptyListView) {
```

Comment:
I assume this is for Dialogs as well, right? Will you have access to the View while creating the initializer or should this be done in the init call?